### PR TITLE
Add cargo-process-run-bin for running specific binaries

### PR DESCRIPTION
**Pull request by TyOverby**

### CODE REVIEW

File: cargo-process.el [hunk 1]
```diff
@@ -283,6 +283,15 @@ Cargo: Build and execute src/main.rs."
   (cargo-process--start "Run" "cargo run"))
 
 ;;;###autoload
+(defun cargo-process-run-bin (command)
+  "Run the Cargo run command --bin <name>.
+With the prefix argument, modify the command's invocation.
+Cargo: Build and execute with --example <name>."
```
Comment (kwrooijen) [inline cargo-process.el]: This line is incorrect, it should be:
Cargo: Run the binary target of the local package

**Review by kwrooijen — CHANGES_REQUESTED** [review]
Make sure you add this command to the comment list:
https://github.com/TyOverby/cargo.el/blob/e07912bb4e07944f8a3fa74bbc331432a87183fc/cargo-process.el#L43

**Comment by kwrooijen** [comment]
Thanks for the PR! There were a couple of things that need to be fixed.

**Comment by TyOverby** [comment]
^ Should be good!

**Comment by kwrooijen** [comment]
LGTM! Thanks!

**Comment by TyOverby** [comment]
🎈 🎈 🎈 🎈

**Pull request merged**